src/components/BrowseSkill/BrowseSkill.js

```diff
@@ -309,6 +310,15 @@ export default class BrowseSkill extends React.Component {
   };
 
   render() {
+    let sidebarStyle = styles.sidebar;
+    let topBarStyle = styles.topBar;
+    let groupsMobile = null;
+    if (window.screen.width < 400) {
```

Comment:
Can you replace `window.screen.width` with `window.innerWidth` so that it will work even when the browser is resized.